util/systemd/rustic-forget@.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=rustic --use-profile %I forget
+
+[Service]
+KillSignal=SIGINT
+ExecStart=/usr/bin/rustic --use-profile %I forget
```

Comment:
I would add `--prune` here. Usually just forgetting without pruning doesn't help that much...

Reply:
Doesn't it make most sense to let people put that in their configs?

Reply:
Ah, of course you are right. The `prune = true/false` option in the config wasn't always present, but now that it is, it makes much more sense putting it there.